package.json

```diff
@@ -14,13 +14,12 @@
     "lint": "eslint *.js src/ config/ --fix",
     "test": "npm run test:unit && npm run test:e2e",
     "pretest:e2e": "npm run build",
-    "test:e2e": "protractor config/protractor.conf.js",
+    "test:e2e": "npm run webdriver && protractor config/protractor.conf.js",
```

Comment:
Instead of calling the webdriver task here you should have used the `ci` task. Because running the webdriver task every time you run the e2e task will make development really slow.